app/src/main/java/com/futurice/freesound/viewmodel/DisplayableItem.java

```diff
@@ -0,0 +1,46 @@
+package com.futurice.freesound.viewmodel;
```

Comment:
This class shouldn't be in this package if it has feature specific types in it - such as `public static final int SOUND = 0;` Otherwise features such as ads etc would touch to many areas. It should either go into `com.futurice.freesound.feature.search` or be generified.